### DISCUSSION TITLE
Custom Order In Files Changed Tab

### DISCUSSION THREAD

**Discussion by glitchedmob** [community/community#70055] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

When submitting a Pull Request I'd like to be able to set the order of the files in the "Files changed" tab instead of it being alphabetical. The use case being that I want a reviewer to have the context of a change (like a new class) before seeing it used/referenced in other files.

**Comment by kd-at-mi** [comment]
This is an amazing suggestion... Workflow++

Suggestions on implementation or how it would work for your workflow?
